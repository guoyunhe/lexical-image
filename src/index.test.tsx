import * as members from '.';

test('members match snapshot', async () => {
  expect(members).toMatchSnapshot();
});
