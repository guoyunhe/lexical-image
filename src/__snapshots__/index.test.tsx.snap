// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`members match snapshot 1`] = `
{
  "ImagePlugin": [Function],
}
`;
